collections/_api_standards/5-api-governance.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title:  "API Governance"
+category: api_standards
+---
+
+## Governance
+
+The AS does not wish to implement a heavyweight API governance and review process, as we feel this will inhibit the ability of the agile teams to deliver business value quickly.  However, we do need to balance the needs of agile, with the needs of quality and consistency.
+
+If you plan to deliver a new API, please contact the AS quality guild.  You will be asked to come and present an outline of your new API and answer questions related to the standards defined above.  

```

Comment:
I don't think this should be a presentation to the quality guild. Some people will just not mention it because they don't want to do that. I think this should be changed to be agreed with the architecture group. This could result in us then saying to the archs that it needs to be presented at the quality guild but will be less of an impact for devs.

Reply:
I don't want Devs to feel that there is a barrier for entry in creating an API. Otherwise they will just default to sending a message as its easier!

Reply:
As requested, changes made to make it clear delivery team engage with architects re APIs (as opposed to quality guild).